cacheServer/PullData.bash

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+
+
+# Config - Which collections to copy over
+collections='campus events posts users ministries summermissions ministryteams'
+
+
+# First, download the JSON for each collection from the official cru database
+echo " --- Downloading JSON ---"
+mkdir -p JSON
+rm -rf JSON/*
+for collection in $collections
+do
+    mongo ds049754.mongolab.com:49754/heroku_s75tvv20 -u capstone-class -p capstone-2015-2016 --quiet --eval "function printResult(r) { print(tojson(r)); } db.$collection.find().forEach(printResult);" > "JSON/$collection.json"
```

Comment:
long line - make multiple lines
              - make function a variable to just plug in